update_prevalence.py

```diff
@@ -250,18 +250,20 @@ class RomaniaPrevalenceData(pydantic.BaseModel):
     TotalCases: int = pydantic.Field(alias="Cazuri total")
 
 
-# Canada Health Region population info dataset:
-class CanadaOpenCovidRegions(pydantic.BaseModel):
-    class RegionInfo(pydantic.BaseModel):
-        HR_UID: int
-        province: str
-        province_full: str
-        health_region: str
-        pop: Union[int, str]
+# Covid Timeline Canada region info dataset:
+class CovidTimelineCanadaRegion(pydantic.BaseModel):
+    SOURCE: ClassVar[str] = "https://raw.githubusercontent.com/ccodwg/CovidTimelineCanada/main/geo/health_regions.csv"
```

Comment:
Health region metadata is no longer hosted via an API, but the data is still around in a CSV we can pull.